modules/exploits/multi/http/october_upload_bypass_exec.rb

```diff
@@ -0,0 +1,159 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'October CMS Upload Protection Bypass Code Execution',
+      'Description' => %q{
+          This module exploits an Authenticated user with permission to upload and manage media contents can
+          upload various files on the server. Application prevents the user from
+          uploading PHP code by checking the file extension. It uses black-list based
+          approach, as seen in octobercms/vendor/october/rain/src/Filesystem/
+          Definitions.php:blockedExtensions().
+          This module was tested against October Machine on HachTheBox.
```

Comment:
I presume you meant `HachTheBox`.

It's great that you tested the module, but the description doesn't really help anyone. The HackTheBox host could change at any time, and there's no way for anyone to host their own, or what the host looked like at a point in time.

Suggest: `This module was tested on October CMS version <version> on Ubuntu.`

Also, not that I care, but some people would consider this to be a spoiler.

Reply:
yup... its hackthebox ... this machine already got retired and available solution on the internet. that's y its not spoiler anymore.